packages/ttypescript/src/PluginCreator.ts

```diff
@@ -204,14 +209,31 @@ export class PluginCreator {
         //   this happens cause ts-node uses to compile transformers the same config included this transformer
         //   so this stack checks that if we already required this file we are in the reqursion
         if (requireStack.indexOf(modulePath) > -1) return;
+
         requireStack.push(modulePath);
-        const factoryModule: PluginFactory | { default: PluginFactory } = require(modulePath);
+        const factoryModule = require(modulePath);
```

Comment:
Can you specify the type of `factoryModule`
`PluginFactory | { [string]: PluginFactory }`